cabal-ghc92.project

```diff
@@ -1,93 +0,0 @@
-packages:
-         ./
-         ./hie-compat
-         ./shake-bench
-         ./hls-graph
-         ./ghcide
-         ./hls-plugin-api
-         ./hls-test-utils
-         ./plugins/hls-tactics-plugin
-         ./plugins/hls-brittany-plugin
-         ./plugins/hls-stylish-haskell-plugin
-         ./plugins/hls-fourmolu-plugin
-         ./plugins/hls-class-plugin
-         ./plugins/hls-eval-plugin
-         ./plugins/hls-explicit-imports-plugin
-         ./plugins/hls-refine-imports-plugin
-         ./plugins/hls-hlint-plugin
-         ./plugins/hls-rename-plugin
-         ./plugins/hls-retrie-plugin
-         ./plugins/hls-haddock-comments-plugin
-         ./plugins/hls-splice-plugin
-         ./plugins/hls-qualify-imported-names-plugin
-         ./plugins/hls-floskell-plugin
-         ./plugins/hls-pragmas-plugin
-         ./plugins/hls-module-name-plugin
-         ./plugins/hls-ormolu-plugin
-         ./plugins/hls-call-hierarchy-plugin
-         ./plugins/hls-alternate-number-format-plugin
-         ./plugins/hls-selection-range-plugin
-         ./plugins/hls-change-type-signature-plugin
-
-with-compiler: ghc-9.2.2
-
-tests: true
-
-package *
-  ghc-options: -haddock
-  test-show-details: direct
-
-write-ghc-environment-files: never
-
-index-state: 2022-04-27T09:22:49Z
-
-constraints:
-  -- These plugins don't build/work on GHC92 yet
-  haskell-language-server
-    +ignore-plugins-ghc-bounds
-    -brittany
-    -haddockComments
-    -retrie
-    -splice
-    -tactic,
```

Comment:
Shouldn't the retrie, splice, and tactics plugins also get `buildable` conditionals?

Reply:
In fact, how is the CI passing?? Did they just start working without us noticing?

Reply:
Oh, it's because we still have the GHC conditionals for inclusion in `haskell-language-server.cabal`

Reply:
Yeah you are right - I'm not sure how `cabal build` decides what to build, so far I'm just adding the `buildable` conditionals on demand